src/main/kotlin/com/terraformation/backend/accelerator/db/AcceleratorOrganizationStore.kt

```diff
@@ -70,4 +72,42 @@ class AcceleratorOrganizationStore(private val dslContext: DSLContext) {
         .fetch { OrganizationModel(it) to it[projectsMultiset] }
         .toMap()
   }
+
+  fun findAllWithProjectApplication(): Map<OrganizationModel, List<ExistingProjectModel>> {
+    requirePermissions { readAllAcceleratorDetails() }
+
+    val hasProjectApplicationCondition =
+        DSL.exists(
+            DSL.selectOne()
+                .from(APPLICATIONS)
+                .join(PROJECTS)
+                .on(PROJECTS.ID.eq(APPLICATIONS.PROJECT_ID))
+                .where(PROJECTS.ORGANIZATION_ID.eq(ORGANIZATIONS.ID)))
+
+    return fetchWithProjects(orgConditions = listOf(hasProjectApplicationCondition))
+  }
+
+  private fun fetchWithProjects(
+      projectConditions: List<Condition> = emptyList(),
+      orgConditions: List<Condition>
```

Comment:
If you flip this so the required parameter comes first, you won't have to have `orgConditions =` at the call site.